internal/pkg/variable/cli.go

```diff
@@ -9,6 +9,76 @@ import (
 	"github.com/zclconf/go-cty/cty"
 )
 
+func (p *Parser) parseEnvVariable(name string, rawVal string) hcl.Diagnostics {
+	// Split the name to see if we have a namespace CLI variable for a child
+	// pack and set the default packVarName.
+	splitName := strings.SplitN(name, ".", 2)
+	packVarName := []string{p.cfg.ParentName, name}
+
+	switch len(splitName) {
+	case 1:
+		// Fallthrough, nothing to do or see.
+	case 2:
+		// We are dealing with a namespaced variable. Overwrite the preset
+		// values of packVarName.
+		packVarName[0] = splitName[0]
+		packVarName[1] = splitName[1]
+	default:
+		// We cannot handle a splitName where the variable includes more than
+		// one separator.
+		return hcl.Diagnostics{
+			{
+				Severity: hcl.DiagError,
+				Summary:  "Invalid NOMAD_PACK_var option",
+				Detail:   fmt.Sprintf("The given environment variable NOMAD_PACK_var_%s=%s is not correctly specified. The variable name must not have more than forward-slash separators.", name, rawVal),
```

Comment:
```suggestion
				Detail:   fmt.Sprintf("The given environment variable NOMAD_PACK_var_%s=%s is not correctly specified. The variable name must not have more than one period separator.", name, rawVal),
```
I "think" you meant this? Also, remind me, will we ever?

Reply:
Ugh... copypasta.  Taking this moment to use the constant in the message in case we decide at a future date to change it.